libs/sdk/e2e-schematics/src/lib/send-cypress-screenshots-to-percy/send-cypress-screenshots-to-percy.ts

```diff
@@ -0,0 +1,108 @@
+import { join, normalize } from '@angular-devkit/core';
+import { stripIndent } from '@angular-devkit/core/src/utils/literals';
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+import utils from '@percy/sdk-utils';
+
+import { readFileSync } from 'fs';
+
+export function sendCypressScreenshotsToPercy(
```

Comment:
This function consolidates a pattern we used in a couple libraries and adds test coverage.